invenio_oauthclient/contrib/keycloak/helpers.py

```diff
@@ -46,22 +45,25 @@ def get_user_info(remote, resp_token,
         endpoint mechanism when verifying the 'id_token' fails
     :param options: A dictionary with additional options for `jwt.decode`
     """
+    app_name = remote.name
+    config_prefix = f"OAUTHCLIENT_{app_name.upper()}"
```

Comment:
should we maybe add some sanitation to `app_name`?
i'm thinking because it could be harder to configure via `invenio.cfg`, if the name contains characters that are invalid for python variable names, e.g. whitespaces or dashes

i think it could still be configured via `app.config["OAUTHCLIENT_MY-KEYCLOAK_AUD"]`, etc., but it would be a bit harder to do
the trade-off would likely be that we have to precisely document this behaviour (something like https://click.palletsprojects.com/en/7.x/options/#name-your-options)

Reply:
That's why I don't like these hidden conventions... I think you are right, I will add some checks